packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.tsx

```diff
@@ -100,16 +98,14 @@ const DateColumnFilter = (props: {
         />
         <KeyboardDatePicker
           style={{ whiteSpace: 'nowrap' }}
-          clearable
-          inputProps={{ 'aria-label': `${props.label} filter to` }}
-          aria-hidden="true"
-          label="To...   [yyyy-MM-dd]"
+          inputProps={{ 'aria-label': `${props.label} filter` }}
```

Comment:
@sam-glendenning @AkhilMag13 as mentioned above, I am adding the "to" back here as well.